Localize/localize-pipeline.yml

```diff
@@ -39,10 +47,10 @@ stages:
         git merge origin/master
         git push origin Localization
       displayName: "Sync with master branch"
-      condition: and(succeeded(), or(and(eq(variables['WEEK'], '3'), eq(variables['build.reason'], 'Schedule')), eq(variables['build.reason'], 'Manual')))
+      condition: and(succeeded(), in(variables['build.reason'], 'Schedule', 'Manual'))
 
     - task: OneLocBuild@2
-      condition: and(succeeded(), or(and(eq(variables['WEEK'], '3'), eq(variables['build.reason'], 'Schedule')), eq(variables['build.reason'], 'Manual')))
+      condition: and(succeeded(), in(variables['build.reason'], 'Schedule', 'Manual'))
```

Comment:
Could you please add shouldCreatePR variable as condition for PR creation by OneLocBuild also? It would be also great to have ability to disable flow with PR creation manually (some additional variable).

Reply:
Added _shouldCreatePR_ as value for _isCreatePrSelected_ input.
And also added an option to disable the PR creating process using PR_CREATION_ENABLED variable in the pipeline.